### PR TITLE
Migrate 'components/admin_console/revoke_token_button' module and associated tests to TypeScript

### PR DESCRIPTION
**Pull request by scottjr632**
**Summary**
This PR migrated 'components/admin_console/revoke_token_button' module and associated tests to TypeScript

**Issue Link**
Resolves: mattermost/mattermost-server#13501

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -3525,7 +3525,7 @@
       "resolved": "https://registry.npmjs.org/@types/redux-mock-store/-/redux-mock-store-1.0.1.tgz",
       "integrity": "sha512-1egEnh2/+sRRKImnCo5EMVm0Uxu4fBHeLHk/inhSp/VpE93It8lk3gYeNfehUgXd6OzqP5LLA9kzO9x7o3WfwA==",
       "requires": {
-        "redux": "^4.0.4"
```
Comment (devinbinnie) [inline package-lock.json]: Can you revert these `package-lock.json` changes? They aren't needed for your PR.

Comment (scottjr632) [reply package-lock.json]: Whoops, sorry. Didn't realize that got committed.

Reverted the change
packge-lock.json is not longer part of the PR.

**Comment by stylianosrigas** [comment]
@lindy65 Cluster is fixed now. Feel free to remove any of the test installations that you don't need. Please let me know if you face any other issues.

**Comment by lindy65** [comment]
Thanks for trying to help spin up a test server @stylianosrigas - seems to still be an issue. The old test server spun up but credentials are not working...

**Pull request merged**